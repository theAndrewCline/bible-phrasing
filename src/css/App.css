@import url('https://fonts.googleapis.com/css?family=PT+Serif:400,400i,700');

h1 {
  color: #006064;
}

.App {
  text-align: center;
  font-family: 'PT Serif', 'serif';
}
