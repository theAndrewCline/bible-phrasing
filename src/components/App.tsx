import React, { Component } from 'react'
import NavBar from './NavBar'
import '../css/App.css'
import { Provider } from 'react-redux'
import { Store } from 'redux'
import { fetchPassage } from '../js/actions/actions';

// const store: Store = createStore()

class App extends Component<{}, {passages: any[]}> {
  constructor (props: any) {
    super(props)

    this.state = {
      passages: []
    }
  }

  public componentWillMount () {
    fetchPassage('John+3').then(passages => { 
      this.setState({passages})
    })
  }
  public render () {
    return (
      // <Provider store={store}>
      <div className='App'>
        <NavBar />
        <h1>John 1</h1>
        {this.state.passages.map((passage, i) => <p key={i}>{passage}</p>)}
      </div>
      // </Provider>
    )
  }
}

export default App
