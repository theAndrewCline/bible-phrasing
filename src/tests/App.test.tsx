import React from 'react'
import App from '../components/App'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('<App />', () => {
  it('should let me commit', () => {
    expect(1).toEqual(1)
  })
  // it('should match snaptshot', () => {
  //   const wrap = shallow(<App />)
  //   expect(toJson(wrap)).toMatchSnapshot()
  // })
})
