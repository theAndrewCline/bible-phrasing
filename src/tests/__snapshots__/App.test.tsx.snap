// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snaptshot 1`] = `
<div
  className="App"
>
  <NavBar />
  <h1>
    John 1
  </h1>
</div>
`;
